game-core/src/main/java/games/strategy/engine/lobby/moderator/toolbox/tabs/access/log/AccessLogTabModel.java

```diff
@@ -0,0 +1,61 @@
+package games.strategy.engine.lobby.moderator.toolbox.tabs.access.log;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.triplea.http.client.moderator.toolbox.PagingParams;
+import org.triplea.http.client.moderator.toolbox.access.log.ToolboxAccessLogClient;
+import org.triplea.http.client.moderator.toolbox.banned.name.ToolboxUsernameBanClient;
+import org.triplea.http.client.moderator.toolbox.banned.user.ToolboxUserBanClient;
+import org.triplea.http.client.moderator.toolbox.banned.user.UserBanParams;
+
+import com.google.common.annotations.VisibleForTesting;
+
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+class AccessLogTabModel {
+
+  @VisibleForTesting
+  static final String BAN_NAME_BUTTON_TEXT = "Ban Name";
+  @VisibleForTesting
+  static final String BAN_USER_BUTTON_TEXT = "Ban User";
+
+  private final ToolboxAccessLogClient toolboxAccessLogClient;
+  private final ToolboxUserBanClient toolboxUserBanClient;
+  private final ToolboxUsernameBanClient toolboxUsernameBanClient;
+
+  static List<String> fetchTableHeaders() {
+    return Arrays.asList(
```

Comment:
Column header definitions were moved to model. This made it easier to line up the column values with the headers.